### ISSUE TITLE
50m Land and 50m Adm0 have different land

### ISSUE THREAD

**Issue by pinakographos** [nvkelso/natural-earth-vector#636]
In a few spots, such as the Lesser Antilles, there are more polygons in the 50m ADM0 layer than appear in the 50m Land file. Maybe this discrepancy doesn't come up too often, but I was using the 50m countries file just for getting some labels, and it was trying to label some places that didn't show up on my land file.

![image](https://user-images.githubusercontent.com/5448396/143129591-c649cc93-3392-4fe8-bdcf-15d5b49d267f.png)

**Comment by nvkelso** [comment]
Are they in the scale rank version?

**Comment by pinakographos** [comment]
Yes, it appears they are. The above was just the entire 50m land (green) atop the entire 50m countries (pink). I just downloaded the 50m scale rank file and these particular islands are all in that file (at varying ranks). There's some inconsistency — some of the islands shown in the 50m land are rank 3, but some of the ones missing from 50m land are also rank 3.
